client/spa/js/learning-resource/learning-resource.controller.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+var Backbone = require('../vendor/index').Backbone;
+var $ = require('../vendor/index').$;
+var Model = require('./learning-resource.model');
+var View = require('./learning-resource.view');
+
+module.exports = Backbone.Controller.extend({
+
+  routes: {
+    'learning-resource/:': 'list',
+    'learning-resource/:id': 'showLearningResource'
+  },
+
+  initialize: function(){
+    this.options.container = this.options.container || 'body';
+    this.model = new Model();
+    this.view = new View({model: this.model});
+  },
+
+  // list: function(){
+  //   this.resourceList = new Model();
+  //   this.resourceListView = new View();
+  //   this.resourceList.fetch();
+  //   $('body').html(this.resourceListView.render.el);
+  // },
+
+  showLearningResource: function(learningResourceId, cb){
+    this.fetchModel(learningResourceId, function(err, model){
+      var view;
+
+      if (err){
+        view = this.renderError();
+      } else {
+        // this.view.destroy();
```

Comment:
I think this was supposed to be `this.view.remove()`.  Calling remove might be a good idea because it cleans up events bound to the old view.